run.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+import os, sys
+
+LOCALPATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, LOCALPATH + '/../../../')
+
```

Comment:
@danuluma What's the purpose of line 4 & 5?

Reply:
it's similar to the above ......